### DISCUSSION TITLE
Implement shared_font_new.bin?

### DISCUSSION THREAD

**Discussion by 20008gui** [skyline-emu/skyline#1629] [Ideas]
Do you think the shared_font_new.bin import used in Egg NS Version 1.0.7 would be good?

The Yuzu Team Permission to Implement This Code?

It would be cool to put this file in the Skyline Menu along with Title.keys and Prod.keys to improve the performance of the games.

**Comment by s3lverd** [comment]
There will be no title and prod keys added as that is illegal if they are not from your own switch.

**Reply by 20008gui** [reply]
But the titles and production keys will not be added, because they are already in the menu to add as soon as i put the game, i'm referring to put the shared_font_new.bin in the menu for agent to add the file in the menu and see what happens in the games
See the assembled example:

Keys

Production Keys | (prod.keys)

Title Keys | (title.keys)

Shared Font New | (shared_font_new.bin)
